packages/dds/map/src/mapKernel.ts

```diff
@@ -748,6 +771,12 @@ export class MapKernel implements IValueTypeCreator {
         this.pendingKeys.set(op.key, pendingMessageId);
     }
 
+    private rebaseMapKeyMessage(op: IMapKeyOperation): void {
+        const pendingMessageId = ++this.pendingMessageId;
+        // this.submitMessage(op, pendingMessageId);
```

Comment:
Unrelated to this code, but a though based on a comment:
For validation of correctness, it would be great for ContainerRuntime / PendingManager to have asserts in place in case a message is being sent during local container rehydration process, as any message sent would derail whole process.